src/Symfony/Component/VarDumper/Dumper/ContextProvider/SourceContextProvider.php

```diff
@@ -96,7 +96,7 @@ public function getContext(): ?array
         $context = ['name' => $name, 'file' => $file, 'line' => $line];
         $context['file_excerpt'] = $fileExcerpt;
 
-        if (null !== $this->projectDir) {
+        if (null !== $this->projectDir && strlen($this->projectDir) > 0) {
```

Comment:
This should be `null !== $this->projectDir && '' !== $this->projectDir` instead.